scipy/stats/morestats.py

```diff
@@ -1083,6 +1100,23 @@ def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
         'all'
             Use all optimization methods available, and return all results.
             Useful to compare different methods.
+    optimizer : callable, optional
+        If `lmbda` is None, `optimizer` is the scalar optimizer used to find
+        the value of `lmbda` that minimizes the negative log-likelihood
+        function. `optimizer` is a callable that accepts one argument:
```

Comment:
```suggestion
        `optimizer` is a callable that accepts one argument:
```
This statement "If `lmbda` is None..." is not needed here; it was erroneously copied from `boxcox`.